packages/typespec-test/test/contentsafety_modular/generated/typespec-ts/src/models/models.ts

```diff
@@ -115,16 +125,32 @@ export interface TextBlockItem {
   text: string;
 }
 
+/** The request of removing blockItems from text blocklist. */
+export interface RemoveBlockItemsOptions {
+  /** Array of blockItemIds to remove. */
+  blockItemIds: string[];
+}
+
 /** Paged collection of TextBlocklist items */
-export interface CustomPage {
+export interface TextBlocklistResult {
```

Comment:
This is the part where the generated code for paged result is CustomPage, and currently just manual give it a name. Note this is not a problem for RLC. so I think the fix should be very simple.